Makefile.am

```diff
@@ -161,13 +161,13 @@ src_tpm2_send_command_SOURCES = src/tpm2_send_command.c src/main.c
 src_tpm2_startup_SOURCES = src/tpm2_startup.c src/main.c
 
 if UNIT
-test_tpm2_rc_decode_unit_CFLAGS  = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
-test_tpm2_rc_decode_unit_LDADD   = $(CMOCKA_LIBS)
-test_tpm2_rc_decode_unit_SOURCES = test/tpm2-rc-decode_unit.c
+test_unit_tpm2_rc_decode_unit_CFLAGS  = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
+test_unit_tpm2_rc_decode_unit_LDADD   = $(CMOCKA_LIBS)
+test_unit_tpm2_rc_decode_unit_SOURCES = test/unit/tpm2-rc-decode_unit.c
 
-test_tpm2_rc_entry_unit_CFLAGS   = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
-test_tpm2_rc_entry_unit_LDADD    = $(CMOCKA_LIBS)
-test_tpm2_rc_entry_unit_SOURCES  = src/rc-decode.c test/tpm2-rc-entry_unit.c
+test_unit_tpm2_rc_entry_unit_CFLAGS   = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
+test_unit_tpm2_rc_entry_unit_LDADD    = $(CMOCKA_LIBS)
+test_unit_tpm2_rc_entry_unit_SOURCES  = src/rc-decode.c test/unit/tpm2-rc-entry_unit.c
```

Comment:
At some point, not now, but these unit test should be auto generated, ie drop in a C file into test/unit and it should just get picked up. There should be no need to write a main, a script could easily scub all matching prototypes into main. Also, the run method for cmock is complaining about deprecated usage.